### PR TITLE
deltachat: Add version 1.3.4

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/deltachat.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.3.4",
+    "description": "Email-based secure instant messaging",
+    "homepage": "https://delta.chat",
```
Comment (Ash258) [inline bucket/deltachat.json:4]:
```suggestion
    "homepage": "https://delta.chat/en/",
```

image

File: bucket/deltachat.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            "hash": "9f6f1db789a8ca2a59e2c6c7423d6d128892b4ec038537a4a3cc22f1110d8b20",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse"
+                ]
+            }
+        }
+    },
+    "bin": "DeltaChat.exe",
```
Comment (Ash258) [inline bucket/deltachat.json:18]: What are available parameters??

Comment (linsui) [reply bucket/deltachat.json:18]: None. Just for consistence with other manifest.

File: bucket/deltachat.json (hunk 1 above)
Comment (linsui) [reply bucket/deltachat.json:4]: I guess it is redirected to different url based on the location. Users can see the page in their languages.

File: bucket/deltachat.json (hunk 2 above)
Comment (Ash258) [reply bucket/deltachat.json:18]: Then remove it.

File: bucket/deltachat.json (hunk 1 above)
Comment (Ash258) [reply bucket/deltachat.json:4]: Use english

**Comment by kunago** [comment]
I used a slightly different approach. Instead of unpacking the DeltaChat.exe file I left it the way it was, using a portable version. Upon start a folder called "DeltaChatData" is created which stores all data. Then it is easy to make the folder persistent and keep the data separately.

Feel free to see https://bitbucket.org/kunago/scoop-bucket/src/master/bucket/deltachat.json

**Comment by linsui** [comment]
@kunago Thanks! I didn't notice that it's portable.

**Comment by Ash258** [comment]
Its not. After launch it is extracted into appdata/local temp and then executed from it. As all electron "portable" binaries.

**Comment by Ash258** [comment]
https://github.com/lukesampson/scoop-extras/pull/2087#issuecomment-488344372

image

**Comment by linsui** [comment]
@Ash258 Yeah, the drawback is it's slow...

**Comment by kunago** [comment]
So far I found it the best solution but if someone will come up with something that can persist the database, then that would be nicer.

**Pull request merged**